mac-setup/build-app.sh

```diff
@@ -16,34 +24,33 @@ cd "${0%/*}" || exit
 
 # delete old app, if there
 echo "clean old app"
-
-export PATH="$HOME/.local/bin:$1/inst/bin:$PATH"
-
 rm -rf ./Xournal++.app
-rm ./Xournal++.zip
 
 echo "prepare gtk-mac-bundler"
 if [ ! -d "gtk-mac-bundler" ]; then
   git clone https://gitlab.gnome.org/GNOME/gtk-mac-bundler.git
-  cd gtk-mac-bundler || exit
-else
-  cd gtk-mac-bundler || exit
-  git pull
+  pushd gtk-mac-bundler
+  make install
+  popd
 fi
 
-make install
-cd ..
+# add gtk-mac-bundler to PATH
+export PATH="$PATH":"$HOME"/.local/bin
```

Comment:
`PATH` is already exported, so the `export` is redundant

Reply:
Doesn't hurt to be explicit here.